packages/airnode-utilities/src/logging.ts

```diff
@@ -0,0 +1,133 @@
+import { formatDateTimeMs } from './utils';
+import { LogLevel, LogOptions, PendingLog, LogMetadata, LogConfig } from './types';
+
+const logLevels: { readonly [key in LogLevel]: number } = {
+  DEBUG: 0,
+  INFO: 1,
+  WARN: 2,
+  ERROR: 3,
+};
+
+export const logger = {
+  buildBaseOptions: (config: LogConfig, meta: LogMetadata) => ({
```

Comment:
Maybe this could be a regular function outside the logger object?

Reply:
That makes sense :+1: